### PR TITLE
Add category resource

### PR DESCRIPTION
**Pull request by FieryCod**
https://tree.taiga.io/project/fierycod-knit-front-vuejs/task/169

### CODE REVIEW

File: store/view/articles.js [hunk 1]
```diff
@@ -119,8 +119,8 @@ export const actions = {
       opus.call(({ ctx }) => ctx.commit(loading(true))),
     ],
     success: [
-      opus.call(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'mainList', codes: result })),
-      opus.call(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'all', codes: result })),
+      opus.call(({ ctx, result }) => ctx.commit(codes({ path: 'mainList', codes: result }))),
+      opus.call(({ ctx, result }) => ctx.commit(codes({ path: 'all', codes: result }))),
```
Comment (LaYatiX) [inline store/view/articles.js]: Why did you change from dispatch to commit?

File: store/view/articles.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const { composer, opus } = storeHelper.opera
 
 // Action names
 const actionNames = {
-  GET_CATEGORY_LIST: '#getCategoryList',
+  GET_CATEGORY_ARTICLES_LIST: '#getCategoryArticlesList',
```
Comment (k911) [inline store/view/articles.js]: const please

File: pages/artykuly/_categoryCode/index.vue [hunk 3]
```diff
@@ -35,19 +35,22 @@ export default {
...
+      return _.map(this.categoryArticlesCodeList, code => this.$store.getters['resources/articles'][code])
     },
   },
   fetch ({ store, params }) {
-    if (params.categoryCode === 'wszystkie') {
-      return store.dispatch('view/articles/getMainList')
-    }
+    const promises = [
+      store.dispatch('view/categories/getCategories'),
+      params.categoryCode === 'wszystkie' ?
```
Comment (k911) [inline pages/artykuly/_categoryCode/index.vue]: 'wszystkie' as const CATEGORY_CODE_ALL ?

Comment (FieryCod) [reply pages/artykuly/_categoryCode/index.vue]: This will be changed in PR which Kamil Stasiak prepares. We managed to not use if in that area at all ;)

Comment (FieryCod) [reply pages/artykuly/_categoryCode/index.vue]: it*
